src/components/DashboardResultIndicatorsSection.vue

```diff
@@ -332,6 +321,28 @@ export default {
       }
     },
 
+    /**
+     * Return a current goal for the active RI if any, else return
+     * null.
+     */
+    async getGoal() {
+      const ri = this.getActiveRI();
+      const now = new Date();
+
+      let goals = await db.collection(`kpis/${ri.id}/goals`)
+          .where('toDate', '>', now)
+          .orderBy('toDate')
+          .get();
```

Comment:
Gjøre en "bind" for `goals` også, som `progressCollection`, slik at vi "abonnerer" på endringer?

Reply:
Lurte på om det funker når vi gjør halvparten av filtreringen på klientsiden her? Eller tenkte du på å binde `goals` uansett, lytte på endringer der, og oppdatere `goal` når det skjer?

Reply:
Hm, var det ja 🤔 Kan kanskje filtreres i en computed?

Reply:
Computed funka ikke, men fikk det til med en watcher!